### PR TITLE
Prepare 3.0.1

### PR DESCRIPTION
**Pull request by ralphtheninja**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,15 @@
...
 ## [Unreleased]
 
+## [3.0.1] - 2018-05-24
+
+### Changed
+* Update `airtap` to `0.0.6` (@ralphtheninja)
+
+### Removed
+* Remove `.jshintrc` (@ralphtheninja)
+* Remove `brfs` and use `Buffer.from()` in favor of `fs.readFilesync()` (@ralphtheninja)
```
Comment (ralphtheninja) [inline CHANGELOG.md]: Just noticed a typo in there, `fs.readFileSync()`

**Comment by ralphtheninja** [comment]
Wtf, still get that error:

```
- starting: <firefox 60 on Windows 10>
2018-05-24T15:13:59.300Z airtap:control-app zuul client took 1.489 seconds to bundle
events.js:167
      throw er; // Unhandled 'error' event
      ^
Error: ENOSPC: no space left on device, watch '/home/travis/build/Level/[secure]/package.json'
    at FSWatcher.start (fs.js:1407:26)
    at Object.fs.watch (fs.js:1444:11)
    at createFsWatchInstance (/home/travis/build/Level/[secure]/node_modules/chokidar/lib/nodefs-handler.js:37:15)
    at setFsWatchListener (/home/travis/build/Level/[secure]/node_modules/chokidar/lib/nodefs-handler.js:80:15)
    at FSWatcher.NodeFsHandler._watchWithNodeFs (/home/travis/build/Level/[secure]/node_modules/chokidar/lib/nodefs-handler.js:228:14)
    at FSWatcher.NodeFsHandler._handleFile (/home/travis/build/Level/[secure]/node_modules/chokidar/lib/nodefs-handler.js:255:21)
    at FSWatcher.<anonymous> (/home/travis/build/Level/[secure]/node_modules/chokidar/lib/nodefs-handler.js:473:21)
    at FSReqWrap.oncomplete (fs.js:183:5)
Emitted 'error' event at:
    at FSWatcher.emit (events.js:182:13)
    at FSWatcher._handleError (/home/travis/build/Level/[secure]/node_modules/chokidar/index.js:257:10)
    at createFsWatchInstance (/home/travis/build/Level/[secure]/node_modules/chokidar/lib/nodefs-handler.js:39:5)
    at setFsWatchListener (/home/travis/build/Level/[secure]/node_modules/chokidar/lib/nodefs-handler.js:80:15)
    [... lines matching original stack trace ...]
    at FSReqWrap.oncomplete (fs.js:183:5)
```

**Comment by vweevers** [comment]
Still got an `ENOSPC` error :(

**Comment by ralphtheninja** [comment]
Ok, so it's not entirely solved in `airtap` and needs further investigation. I'll restart that build and still merge this.

**Pull request merged**